qa/tasks/cephadm.py

```diff
@@ -7,6 +7,7 @@
 import logging
```

Comment:
"chacra" is misspelled in the commit message; should fix before merge

Reply:
Fixed, thanks.